spec/features/visitor/create_post_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+feature "Create a post" do
+  scenario "Visitor creates a post" do
+    visit new_post_path
+
+    expect(page).to have_content("You are not authorized! Please sign in.")
```

Comment:
As I mentioned before when "not authorized" means that user cannot perform some action (edit/delete post for example). "Sign in" does not make sense in this example.

```suggestion
    expect(page).to have_content("You are not authorized to perform this action!")
```